### PR TITLE
Russian translation major update

### PR DESCRIPTION
**Pull request by R1BLH**
Russian translation update (new lines and fixes)

### CODE REVIEW

**Comment by HB9HIL** [comment]
Hello R1BLH

Please remove the mo file. We compile the mo file directly in Github. 

Details you can see [here](https://github.com/wavelog/wavelog/wiki/Translations). In the meantime I review your translations.

File: application/locale/ru_RU/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -234,27 +235,27 @@ msgstr ""
...
 
 #: application/controllers/Awards.php:467
 msgid " and mode "
-msgstr ""
+msgstr " и режим модуляции"
 
 #: application/controllers/Awards.php:470
 msgid " and "
-msgstr ""
+msgstr " и"
```
Comment (HB9HIL) [inline application/locale/ru_RU/LC_MESSAGES/messages.po:258]: here is missing a space 
```sugestion
msgstr " и "
```

Comment (R1BLH) [reply application/locale/ru_RU/LC_MESSAGES/messages.po:258]: Ok. Thanks.
I'll check in two hours. It's very difficult to do this from a cell phone.

**Comment by HB9HIL** [comment]
Had a chat with @R1BLH 

he will upload his translations to https://translate.wavelog.org/ 
In this way we will have all the quality checks and a structured workflow :)

Thank you @R1BLH !

**Pull request closed without merging**